symfony/stimulus-bundle/2.20/assets/controllers/csrf_protection_controller.js

```diff
@@ -0,0 +1,60 @@
+nameCheck = /^[-_a-zA-Z0-9]{4,22}$/;
+tokenCheck = /^[-_/+a-zA-Z0-9]{24,}$/;
```

Comment:
Good catch, but more `const` or `let` to prevent global access to them

Reply:
Since no `const` or `let` has been used in this file so far, I have also used `var` here ;)

Reply:
I suggest migrating to `const` everywhere in the file, as that's the modern way to write JS code (and Stimulus 3+ does not support old browsers that don't support `const` anyway)

Reply:
I added `var` but I don't think it's needed since we're at the root of the scope
I'm keeping `var` just because it makes the code compatible with even more old browser (eases copy/pastes if one what to put the code outside of a stimulus-loaded module.

Reply:
> I added `var` but I don't think it's needed since we're at the root of the scope 

It's needed, otherwise the variable in globally accessible and mutable by other scripts 

Thanks!